test/test_art.py

```diff
@@ -367,12 +367,7 @@ def test_leave_original_file_in_place(self):
         self.assertExists(self.art_file)
 
     def test_delete_original_file(self):
-        config['import']['delete'] = True
-        self._fetch_art(True)
-        self.assertNotExists(self.art_file)
-
-    def test_move_original_file(self):
-        config['import']['move'] = True
+        self.plugin.src_removed = True
```

Comment:
This obviously takes some detail out of the test. It is however consistent with the way `cautious` is tested, which isn't changed in the config either in `FSArtTest`.
Moving the config reads to FetchArtPlugin__init__ seemed cleaner to me, it makes keeping this test as-is difficult, though